example-testng-fork-execution/src/main/resources/reportportal.properties

```diff
@@ -0,0 +1,11 @@
+rp.endpoint =
+
+rp.uuid = 
+
+rp.launch = superadmin_TEST_EXAMPLE
+
+rp.project = superadmin_personal
```

Comment:
Please apply common format for the fields above.

Reply:
fixed